jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunner.kt

```diff
@@ -56,10 +57,10 @@ open class SamRunner(protected val settings: LocalLambdaRunSettings) {
             processHandler.waitFor()
             val exitValue = processHandler.exitCode
             if (exitValue != 0) {
-                throw Exception(message("lambda.debug.docker.not_connected"))
+                throw ExecutionException(message("lambda.debug.docker.not_connected"))
```

Comment:
can we write-out stderr to help the user out a bit? (so they can tell if docker was not in the PATH, docker internal failure, etc.)

Reply:
Hmmm... We should probably move this to a Step so it has a console to write to?

Reply:
ya, can be a follow-up later

Reply:
Will convert this to a step after #2436 is merged

Reply:
Converted